contributors/design-proposals/aws/OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - sig-aws-leads
+  - provider-aws
 approvers:
-  - sig-aws-leads
+  - provider-aws
 labels:
   - sig/aws
```

Comment:
Would be nice to change the label to `provider/aws` too btw :grimacing:

Reply:
@nikhita -- can't change the labels just yet, as it requires a k/test-infra change. I can sweep those in a follow-up PR.

Reply:
Follow-up sgtm. Created https://github.com/kubernetes/test-infra/pull/13677 to get the ball rolling. :)

Reply:
Thanks @nikhita!